src/config-parser.js

```diff
@@ -56,14 +56,32 @@ class ConfigParser {
   // Return the configuration object or null.
   findConfigurationObject(ast) {
     // Try to find a default export
-    var defaultExport = ast.body.find(
-      node => node.type === 'ExportDefaultDeclaration' && node.declaration.type === 'ObjectExpression'
-    )
-    if (defaultExport) {
-      core.info('Found configuration object in default export declaration')
+    var defaultExport = ast.body.find(node => node.type === 'ExportDefaultDeclaration')
+
+    // Direct default export
+    if (defaultExport && defaultExport.declaration.type === 'ObjectExpression') {
+      core.info('Found configuration object in direct default export declaration')
       return defaultExport.declaration
     }
 
+    // Indirect default export
+    else if (defaultExport && defaultExport.declaration.type === 'Identifier') {
+      const identifierName = defaultExport && defaultExport.declaration.name
```

Comment:
`defaultExport` is already defined in this block

```suggestion
      const identifierName = defaultExport.declaration.name
```